src/espina/shared/application_component.js

```diff
@@ -55,4 +55,6 @@ ApplicationComponent.childContextTypes = {
 	state_manager: React.PropTypes.instanceOf(StateManager).isRequired,
 	router: React.PropTypes.instanceOf(Router).isRequired,
 	i18n: React.PropTypes.object.isRequired
-}
\ No newline at end of file
+}
```

Comment:
I'll have to re-think providing these as a base component. As long as it's not a requirement for user to use the base component we are fine.